scripts/tag_editor_ui/block_dataset_gallery.py

```diff
@@ -22,7 +22,7 @@ def create_ui(self, image_columns):
             self.btn_hidden_set_index = gr.Button(elem_id="dataset_tag_editor_btn_hidden_set_index")
             self.nb_hidden_image_index = gr.Number(value=None, label='hidden_idx_next')
             self.nb_hidden_image_index_prev = gr.Number(value=None, label='hidden_idx_prev')
-        self.gl_dataset_images = gr.Gallery(label='Dataset Images', elem_id="dataset_tag_editor_dataset_gallery").style(grid=image_columns)
+        self.gl_dataset_images = gr.Gallery(label='Dataset Images', elem_id="dataset_tag_editor_dataset_gallery", grid=image_columns)
```

Comment:
Incorrect argument to `gr.Gallery`. Should be `columns=image_columns`, not `grid=image_columns`:
```diff
-self.gl_dataset_images = gr.Gallery(label='Dataset Images', elem_id="dataset_tag_editor_dataset_gallery", grid=image_columns)
+self.gl_dataset_images = gr.Gallery(label='Dataset Images', elem_id="dataset_tag_editor_dataset_gallery", columns=image_columns)
```